features/core-download.feature

```diff
@@ -95,3 +95,98 @@ Feature: Download WordPress
       """
       File removed: wp-content
       """
+
+  Scenario: Installing nightly
+    Given an empty directory
+    And an empty cache
+
+    When I run `wp core download --version=nightly`
+    Then the wp-settings.php file should exist
+    And the {SUITE_CACHE_DIR}/core/wordpress-nightly-en_US.zip file should exist
```

Comment:
Oh, given the cache expiry is set to 6 months, we probably shouldn't be caching the nightly build. Sorry I missed that before :(